src/cosmic_ray/work_item.py

```diff
@@ -84,6 +84,9 @@ def __eq__(self, rhs):
     def __neq__(self, rhs):
         return not self == rhs
 
+    def __repr__(self):
```

Comment:
This is really nice, and something I've intended to do for a while. I'll go ahead and pull this (and WorkItem.__repr__) over to master now so we don't lose it.

Reply:
Small note though: We still maintain python-3.5 compatibility, so we can't use f-strings. Our Travis build will catch this, but it's something to remember for the future.